server/src/main/scala/services/crunch/deskrecs/RunnableDeskRecs.scala

```diff
@@ -57,7 +57,7 @@ object RunnableDeskRecs {
               val crunchEndMillis = SDate(crunchStartMillis).addMinutes(portDeskRecs.minutesToCrunch).millisSinceEpoch
               val minuteMillis = crunchStartMillis until crunchEndMillis by 60000
 
-              log.info(s"Crunching ${flights.flights.size} flights, ${minuteMillis.length} minutes (${SDate(crunchStartMillis).toISOString()} to ${SDate(crunchEndMillis).toISOString()})")
+              log.info(s"**\n\nCrunching ${flights.flights.size} flights, ${minuteMillis.length} minutes (${SDate(crunchStartMillis).toISOString()} to ${SDate(crunchEndMillis).toISOString()})\n\n")
```

Comment:
do we want to keep these new lines in the logs?

Reply:
nope sorry!